keyboards/handwired/dactyl_manuform/5x6/keymaps/default_reversed/keymap.c

```diff
@@ -0,0 +1,49 @@
+/* A standard layout for the Dactyl Manuform 5x6 Keyboard */ 
+
+#include QMK_KEYBOARD_H
+
+
+#define _QWERTY 0
+#define _LOWER 1
+#define _RAISE 2
```

Comment:
Normally we encourage an `enum` here but since the default keymap uses this construct, and it looks like you're just trying to mirror that, I suppose this is fine here.